applications/OptimizationApplication/custom_utilities/container_variable_data_holder_utils.cpp

```diff
@@ -148,6 +148,17 @@ double ContainerVariableDataHolderUtils::NormInf(const ContainerVariableDataHold
     }));
 }
 
+double ContainerVariableDataHolderUtils::NormInf(
```

Comment:
What is the difference between container_variable_data_holder and Eigen::Vector ?

Reply:
`Eigen` has its own lazy expression system  which is constructed at compile time using compile time polymorphism. The `ContainerExpression` and `CollectiveExpressions` does the same but at runtime using runtime polymorphism.